source/docs/03-metadata.xml

```diff
@@ -1382,156 +1382,318 @@
+                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample080.txt" parse="text"/></egXML>
                         </figure>
                      </p>
+                     <p>If those value lists are not sufficient, however, it is recommended to design your own classification (see <gi scheme="MEI">classification</gi>).</p>
+                           <p><gi scheme="MEI">scoreFormat</gi> can similarly be used for classification in <gi scheme="MEI">expression</gi>.</p>
+                              <!--<p>Internal note: Even <gi scheme="MEI">physDesc</gi> is part of the MEI.header module, the parent elements of <gi scheme="MEI">physDesc</gi> (<gi scheme="MEI">manifestation</gi>, <gi scheme="MEI">item</gi>) are part of MEI.frbr. So the use of the module MEI.frbr is mandatory for the use of <gi scheme="MEI">scoreFormat</gi>.</p>-->
                   </div>
-                  <div xml:id="foliumdimensions" type="div4">
-                     <head>Specifying page dimensions</head>
-                     <p>Within <gi scheme="MEI">surface</gi>, each <gi scheme="MEI">graphic</gi> element may specify its dimensions using the following attributes:</p>
+                  <div xml:id="bindingDesc" type="div4">
+                     <head>Binding Description</head>
+                  <p></p>
                      <p>
                         <specList>
-                           <specDesc key="att.height" atts="height"/>
-                           <specDesc key="att.width" atts="width"/>
+                           <specDesc key="bindingDesc"/>
+                           <specDesc key="binding"/>
                         </specList>
                      </p>
-                     <p>The values of those attributes, however, specify the height and width of the digital resource, the scan of the source, and they are typically given in pixels (see <ptr target="#facsimileElements"/>). In contrast, <gi scheme="MEI">folium</gi> and <gi scheme="MEI">bifolium</gi> are supposed to provide the dimensions of the original source in cm or inches. This makes it possible to combine separate parts of a manuscripts stored in different libraries, which are scanned at different resolutions. In case of <gi scheme="MEI">bifolium</gi> elements, these dimensions apply to the <hi rend="italic">folded</hi> sheet.</p>
-                     <p>Some printed scholarly editions like the <hi rend="italic">Neue Bach-Ausgabe</hi> provide very detailed information about the sizes and binding of individual leaves of a manuscript; with <gi scheme="MEI">foliaDesc</gi> and its children it is possible to encode that information in processable ways, even without providing digitizations of the sources via <gi scheme="MEI">surface</gi>.</p>
+                     <p></p>
```

Comment:
I'm not going to continue to point out the empty paragraphs, but they should be removed.

Reply:
I deleted all the placeholders.